github-electron/github-electron-main-tests.ts

```diff
@@ -26,9 +26,6 @@ import path = require('path');
 // Quick start
 // https://github.com/atom/electron/blob/master/docs/tutorial/quick-start.md
 
-// Report crashes to our server.
-require('crash-reporter').start();
-
```

Comment:
https://github.com/hinamiyagk/DefinitelyTyped/blob/crashreporter-start-option/github-electron/github-electron-main-tests.ts#L493-L501
I think it's OK because test already exists.